docs/api/schema-registry.md

```diff
@@ -24,7 +24,7 @@ The following code sample will create 2 schemas in the `Schema Registry`.
 ```python
 import asyncio
 
-from tartiflette import Engine, Resolver
+from tartiflette import create_engine, Resolver
```

Comment:
Correct order is Resolver, create_engine (per isort, Uppercases is before lowercases)